extras/lbf-nix/lbf-prelude-typescript.nix

```diff
@@ -0,0 +1,42 @@
+# Build .lbf schemas that use LB Prelude package and targets TypeScript Prelude
+# (and friends) library.
+# Warning: Essentially duplicated code from  `./lbf-prelude-haskell.nix`
+# TODO(jaredponn): instead of passing everything in, how about we just "go all
+# the way" using flake-parts and make all these scripts their own flake-parts
+# module?
```

Comment:
I don't see them as flakes though? what would be the output config here?

Reply:
Hm, lemme backtrack a bit with what I was thinking... might've not made any sense at all xD.

It  seems to me that we use all of these `lbf-*-*.nix` files only in https://github.com/mlabs-haskell/lambda-buffers/blob/main/extras/lbf-nix/build.nix, which importantly defines the `perSystem` `lbf-nix` option. So, I suppose I was thinking that the output config of each of these `lbf-*-*.nix` files would output a `perSystem` `lbf-nix.<attr>` which would all get merged into a single attribute set. 

I believe that would be equivalent to what already exists, and wouldn't require passing everything into the files.

I may be mistaken!